src/main/java/app/App.java

```diff
@@ -36,6 +38,14 @@ private static void run(String[] args) throws IOException {
                 case "SPARKSQL":
                     importer = new SparkSQLImporter(importerConfig);
                     break;
+                case "HDFS":
```

Comment:
Need to create Instance of Connector class. This code can go into the execute() method of an HDFSImporter class